app/donglet/app-g031.toml

```diff
@@ -96,11 +96,12 @@ address = 0x73
 [[config.i2c.devices]]
 controller = 1
 mux = 1
-segment = 1 
+segment = 1
 address = 0b1010_000
 device = "at24csw080"
 description = "Sharkfin VPD"
 removable = true
+refdes = ["J3", "J3"]
```

Comment:
Where does the second `J3` come from?  I see the EEPROM as `U7` in the Sharkfin schematic